### ISSUE TITLE
In Taskly App

### ISSUE THREAD

**Issue by Ramsthr** [preneure/the_complete_flutter_development_course#1]
There is no intialisation folder for hive.
you initialised in code but in directory we don,t have such folder.

**Comment by dhirajchaurasiya** [comment]
Yes, however, not to get confused it's necessary to make checkbox and delete work so it would be better to keep. That's all I would say for the new learners.
